### PR TITLE
New Brazilian exchange rates

### PR DESCRIPTION
**Pull request by Marcdnd**
Please close pull request # 5286 and consider this!
Thank you!

### CODE REVIEW

File: electrum/exchange_rate.py [hunk 1]
```diff
@@ -184,6 +184,20 @@ class Bitbank(ExchangeBase):
...
+        json = await self.get_json('api.bitvalor.com', '/v1/ticker.json')
+        return {'BRL': Decimal(json['ticker_1h']['exchanges']['CAM']['last'])}
+
+
+class BitcoinTrade(ExchangeBase):
+
+    async def get_rates(self, ccy):
+        json = await self.get_json('api.bitvalor.com', '/v1/ticker.json')
+        return {'BRL': Decimal(json['ticker_1h']['exchanges']['BTD']['last'])}
+
```
Comment (SomberNight) [inline electrum/exchange_rate.py]: you need to also add these two exchanges to `currencies.json`

File: electrum/exchange_rate.py [hunk 2]
```diff
@@ -238,6 +252,20 @@ class BlockchainInfo(ExchangeBase):
         return dict([(r, Decimal(json[r]['15m'])) for r in json])
 
 
+class Brabex(ExchangeBase):
+
+    async def get_rates(self, ccy):
+        json = await self.get_json('exchange.brabex.com.br', 'api/v1/BRL/ticker?crypto_currency=BTC')
```
Comment (SomberNight) [inline electrum/exchange_rate.py]:
```suggestion
        json = await self.get_json('exchange.brabex.com.br', '/api/v1/BRL/ticker?crypto_currency=BTC')
```

Comment (Marcdnd) [reply electrum/exchange_rate.py]: Fixed!

**Comment by deyvisonrocha** [comment]
Thanks a lot! Nice job man!

**Comment by helpau** [comment]
Looks like only walltime is active exchange. And it is already added. Probably need to close this PR and #5286

**Comment by accumulator** [comment]
thanks, closing.

**Pull request closed without merging**